src/components/footer/footer.module.scss

```diff
@@ -0,0 +1,123 @@
+.footer {
+  margin-top: 5rem;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 0;
+}
+
+.firstSection {
+  display: flex;
+
+  .logoSection {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+  .logo {
+    width: 20rem;
+    height: auto;
+  }
+
+  .social {
+    display: flex;
+    justify-content: space-evenly;
+    img {
+      width: 2rem;
+      height: auto;
+    }
+  }
+}
+.secondSection {
+  background-color: #484848;
+  height: 6rem;
+}
+.thirdSection {
+  background-color: #393838;
```

Comment:
I would recommend to use variables for the color theme instead of hardcoding the same hex code every time, even if you are gonna need it once it's a good practice, its easier to look into a single place when you want to update things
- for the theme I main colors I would recommend global variables

Reply:
Color variables were created in the root of globals.css and updated in each path.